ros2topic/ros2topic/verb/delay.py

```diff
@@ -124,7 +123,7 @@ def callback_delay(self, msg):
                                    (msg.header.stamp.sec + msg.header.stamp.nanosec * 1e-9))
                 self.msg_tn = curr
 
-            if len(self.delays) > self.window_size - 1:
+            if len(self.delays) > self.window_size:
```

Comment:
Please describe why this was changed.

Reply:
To fix the issue that no data print when set window as 1, 
```ros2 topic delay /image -w 1```

Reply:
Don't know why here need to window_size - 1 ?